### DISCUSSION TITLE
I'm a bit confused about the general functionality

### DISCUSSION THREAD

**Discussion by mztiq** [cmintey/wishlist#149] [Q&A] [answered]
I created two users (Admin and Tester) for testing purposes, both users have items in their wishlist and are in the same group.
Admin can only see Tester's wishlist as long as it's empty — as soon as I add items in Tester's wishlist, Admin can't see the Wishlist in the Home Tab and vice versa. 

Empty Wishlist:
![empty_wishlist](https://github.com/user-attachments/assets/e79eb023-705e-4d1e-b134-63f831bb88ac)

View of Admin's Home Tab after Tester added an item to his Wishlist:
![item_added_to_tester](https://github.com/user-attachments/assets/c97bd55c-da38-414e-9323-7af1b7c900c9)

Do I really have to access Tester's wishlist directly via the URL or am I missing something here?

**Comment by cmintey** [comment, marked as answer]
Hi, are you just testing using localhost? I think this may be related to a bug I recently found

**Reply by mztiq** [reply]
yes, you're right. I didn't bother setting it up via reverse proxy for testing but I did now and everything works as expected. Thanks!
